website/docs/configuration.md

```diff
@@ -3,8 +3,77 @@ id: configuration
 title: Configuration
 ---
 
-TODO: Cover the mandatory configuration options here.
+Docusaurus asks for only a few fields to configure your site.
 
-#### References
+## `docusaurus.config.js`
 
-- https://v1.vuepress.vuejs.org/guide/basic-config.html
+When you create your site using `docusaurus init`, the `docusaurus.config.js` file is initialized alongside the template.
+
+### Configurable fields
+
+`docusaurus.config.js` contains configurable fields for:
+
+- Meta data of your site
+  - `title: string`
+  - `tagline: string`
+  - `favicon: string` relative url to your site's logo
```

Comment:
it does accept external or even absolute url.

Reply:
its not clear as well what is tagline. and what is the mandatory field. there is no mention of baseurl here.

try reading https://docusaurus.io/docs/en/site-config.html#siteconfig-fields